src/lib/consts.js

```diff
@@ -26,20 +26,20 @@ exports.ACTS_TEMPLATES = {
             memoryMbytes: 2048,
         },
     },
-    puppeteer_crawler: {
-        name: 'Puppeteer crawler ("puppeteer_crawler")',
-        value: 'puppeteer_crawler',
-        dir: `${__dirname}/../templates/puppeteer_crawler`,
+    basic: {
+        name: 'Basic - Minimum boilerplate for an actor without chrome browser',
+        value: 'basic',
+        dir: `${__dirname}/../templates/basic`,
         defaultRunOptions: {
             build: 'latest',
-            timeoutSecs: 0,
-            memoryMbytes: 2048,
+            timeoutSecs: 3600,
+            memoryMbytes: 512,
         },
     },
-    plain_request_urls_list: {
-        name: 'Plain request url list crawler ("plain_request_urls_list")',
-        value: 'plain_request_urls_list',
-        dir: `${__dirname}/../templates/plain_request_urls_list`,
+    hello_word: {
+        name: 'Hello word - Example boilerplate from apify SDK documentation',
```

Comment:
`Hello word - An example actor demonstrating recursive crawling of a website using headless Chrome and Puppeteer.`